google-cloud-datastore/src/test/java/com/google/cloud/datastore/it/ITDatastoreTest.java

```diff
@@ -532,14 +528,11 @@ public void testAllocateIdArray() {
     Key key3 = keyFactory.newKey("name");
     Key key4 = keyFactory.newKey(1);
```

Comment:
Delete `key3` and `key4` since they aren't used any more. (Same thing in other tests too)